modules/social_features/social_post/social_post.module

```diff
@@ -104,6 +107,7 @@ function _social_post_comment_post_comment_form_submit(array $form, FormStateInt
   if (UrlHelper::isValid($uri)) {
     // Get the Url from the Form value and redirect to this url.
     $url = Url::fromUserInput($uri);
+    var_dump($url);
```

Comment:
Hi Brhane, make sure to remove these. 

A good idea is to always check in github what code you have actually pushed to make sure no debug messages land in the finald push.